README.md

```diff
@@ -294,6 +294,26 @@ $ cd android-components
 $ ./gradlew assemble
 ```
 
+### Configuring the Android SDK ###
+
+You will need the Android SDK installed (i.e. pointed to by the `ANDROID_SDK_ROOT` environment variable), and configured to indicate acceptance of the Android SDK licenses. This can be accomplished on Linux as follows:
+
+```
+$ wget http://dl.google.com/android/android-sdk_r24.2-linux.tgz
+$ tar -xvf android-sdk_r24.2-linux.tgz
+$ cd android-sdk-linux
+$ mkdir -p licenses
+$ echo "8933bad161af4178b1185d1a37fbf41ea5269c55" >> licenses/android-sdk-license
+$ echo "d56f5187479451eabf01fb78af6dfcb131a6481e" >> licenses/android-sdk-license
+$ echo "24333f8a63b6825ea9c5514f83c2829b004d1fee" >> licenses/android-sdk-license
+```
+
+Then run `pwd` and use the resulting value as `ANDROID_SDK_ROOT`
+
+### Configuring Java ###
+
+You will need `JAVA_HOME` set to a path to a JDK (which should not end in `/jre`).
```

Comment:
Android Studio also has a packaged version of `OpenJDK` so I think there is something else at play here (see review comment).

Reply:
I tried to put this under the command line, non-Android-Studio heading. I was specifically trying to avoid the Android Studio 800 MB download and get the build working with as few moving parts as possible (i.e. on the system-provided JDK, entirely from the command line, with only the Android SDK components that are actually necessary, and without installing a whole IDE).

In my particular environment (on Ubuntu with Java 8 set as the default Java) I had to make sure to set JAVA_HOME to the Java 11 JDK. I'm not sure if that's because it picked up the Java 8 JDK which was incompatible somehow, or it picked up the Java 8 JRE which wasn't a JDK. (Setting JAVA_HOME to the Java 8 JDK still seemed to produce the error about not being able to find the required JDK tools.)

If it auto-magically detects the JDK for most people (maybe on other distros or when a reasonably new Java is the default) this could definitely be softened to a troubleshooting tip.